src/main/resources/messages/MessageBundle_ru.properties

```diff
@@ -0,0 +1,56 @@
+promotion.action.cancel=No Thanks
```

Comment:
@Dragon-0609 This is the file you'll add translations to. You can probably either edit the file in this pr or suggest one big edit. If that doesn't work, then you'll just have to fork code base checkout the `cheekiBreeki` branch and submit a merge request to this codebase's `cheekiBreeki` branch. Thanks in advance!